modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -281,6 +290,27 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		return Objects.equals(toString(), ${schemaVarName}.toString());
 	}
 
+	<#if generateGetValues?has_content>
+		<#assign firstPropertyName = generateGetValues[0]>
+			public Object getValue(String propertyName) {
```

Comment:
I think the commit and the PR description is wrong. The method is called getValue